apps/onboarding-ms/src/main/java/it/pagopa/selfcare/onboarding/controller/AggregatesController.java

```diff
@@ -38,5 +39,17 @@ public Uni<VerifyAggregateResponse> verifyAppIoAggregatesCsv(@NotNull @RestForm(
     }
 
 
+    @Operation(
+            summary = "Retrieve aggregates csv for a given onboarding and product",
+            description = "Downloads the aggregates csv associated with the specified onboarding ID and product."
+    )
+    @GET
+    @Produces(MediaType.APPLICATION_OCTET_STREAM)
+    @Path("/csv/{onboardingId}/{productId}")
```

Comment:
it might be better to add "products" in the path, for ex: /csv/{onboardingId}/products/{productId}"....this path follow API REST best practices